static/app/stories/view/index.tsx

```diff
@@ -183,7 +183,7 @@ const StoryMainContainer = styled('div')`
     margin: 0;
   }
 
-  code:not(pre > code) {
+  code:not([class]):not(pre > code) {
```

Comment:
huh, I don’t really like that we have global styles applied to our stories. For one, that means they might look different than production - we should really re-use the global production styles for a what-you-see-is-what-you-get behaviour. And two, it might mean our primitives don’t cover those situations, or we are not using our primitives in those places.

Reply:
That's a very good point. The global styles predate our primitives 😅 But we do have an MDX component provider now and could probably remove all of these styles in favor of wrapping with `Prose`. Can tackle that in a follow-up.